Feature/new demo site (#1046)
daneden/animate.css: docs/pages/02-utilities.md

| ------------------- | ---------- |
| `animate__delay-2s` | `2s`       |
| `animate__delay-3s` | `3s`       |
| `animate__delay-4s` | `4s`       |
| `animate__delay-5s` | `5s`       |

The provided delays are from 1 to 5 seconds. You can customize them setting the --animate-delay property to a longer or a shorter duration:

```css
/* All delay classes will take 2x longer to start */
:root {
  --animate-delay: 2s;
}

/* All delay classes will take half the time to start */
:root {
  --animate-delay: 0.5s;
}
```

### Slow, slower, fast, and Faster classes

You can control the speed of the animation by adding these classes, as below:

```html
<div class="animate__animated animate__bounce animate__faster">Example</div>
```

| Class Name        | Speed Time |
| ----------------- | ---------- |
| `animate__slow`   | `2s`       |
| `animate__slower` | `3s`       |
| `animate__fast`   | `800ms`    |
| `animate__faster` | `500ms`    |

The `animate__animated` class has a default speed of `1s`. You can also customize the animations duration through the `--animate-duration` property, globaly or locally. This will affect both the animations and the utility classes. Example:

```css

> /* All animations will take twice longer to finish */

Reviewer: ```suggestion
/* All animations will take twice as long to finish */
```

:root {
  --animation-duration: 2s;
}

/* Only this element will take half the time to finish */
.my-element {
  --animation-duration: 0.5s;
}
```

> Notice that some animations have a duration of less than 1 second. As we used the CSS `calc()` function, setting the duration through the `--animation-duration` property will respect these ratios. So, when you change the global duration all the animations will keep its consistency!

Reviewer: ```suggestion
Notice that some animations have a duration of less than 1 second. As we used the CSS `calc()` function, setting the duration through the `--animation-duration` property will respect these ratios. So, when you change the global duration, all the animations will respond to that change!
```
